nucliadb_models/nucliadb_models/text.py

```diff
@@ -83,9 +83,10 @@ class TextField(BaseModel):
     body: str
     format: TextFormat = TextFormat.PLAIN
 
-    @root_validator(pre=False, skip_on_failure=True)
+    @model_validator(mode="before")
```

Comment:
`mode="after"` ? how is the skip on failure done in pydantic 2?

Reply:
Yeah, it should be mode after. I changed it now. It also means changing the validator, so that's why I didn't change all of them at the first pass and I forgot some, thanks for noticing! :heart:.

`skip_on_failure` is not applicable to `mode=after` since what it does is skip pydantic own validators, which have already run when using mode=after.